conf/uppmax.config

```diff
@@ -7,6 +7,7 @@ params {
 
 singularity {
   enabled = true
+  cacheDir = '/sw/data/uppnex/ToolBox/nf-core'
```

Comment:
```suggestion
```
But you're right, as it's not working now, I'll remove it from this PR, and it'll come in a new one